comp/safety/incident-log.yaml

```diff
@@ -0,0 +1,25 @@
+summary: Update and publish risk assessments for the competition
```

Comment:
This summary seems unrelated to the actual task?

Reply:
[570c0a4](https://github.com/srobo/recurring-tasks/pull/90/commits/570c0a4d355beee50ed64735ae6c2dd91bd76779)